src/library_fs.js

```diff
@@ -400,7 +400,7 @@ FS.staticInit();` +
     // SOCKFS is completed.
     createStream: (stream, fd_start, fd_end) => {
       if (!FS.FSStream) {
-        FS.FSStream = /** @constructor */ function(){};
+        FS.FSStream = /** @constructor */ function(){ this.shared = { refcnt : 0 }; };
```

Comment:
Can you spread this across multiple lines? (and add a space before the opening `{`)